assignment/Main.hs

```diff
@@ -18,13 +22,19 @@ hub =
   circleSolid 40 
   |> color orange 
 
+label seconds = 
+    text format(seconds)
+    |> color red
+
 main = 
   animate windowed azure renderer
     where
       windowed = InWindow "My Window" (800, 800) (10, 10)
       renderer seconds = 
         pictures [
            face,
-           hand seconds,
+           hand (seconds * 30),
+           hand ((seconds * 30) / 60), 
+           label,
```

Comment:
You were also missing the `seconds` parameter on the call to `label`.

I changed your code, and got this:

![image](https://user-images.githubusercontent.com/10648/116441496-1bdb4080-a817-11eb-9baf-f06afadcc8ad.png)